app/web/src/features/auth/Settings.tsx

```diff
@@ -18,31 +19,60 @@ import {
 } from "./constants";
 import useAccountInfo from "./useAccountInfo";
 
+const useStyles = makeStyles((theme) => ({
+  section: {
+    margin: theme.spacing(4, 0),
+    "&:first-of-type": {
+      marginTop: theme.spacing(2),
+    },
+  },
+}));
+
 export default function Settings() {
   const {
     data: accountInfo,
     error: accountInfoError,
     isLoading: isAccountInfoLoading,
   } = useAccountInfo();
 
+  const classes = useStyles();
+
   return (
     <>
       <HtmlMeta title={ACCOUNT_SETTINGS} />
       <PageTitle>{ACCOUNT_SETTINGS}</PageTitle>
-      <Typography variant="h2">{CHANGE_GENDER}</Typography>
-      <Typography variant="body1">{CHANGE_GENDER_CONTACT}</Typography>
-      <Typography variant="h2">{CHANGE_BIRTHDATE}</Typography>
-      <Typography variant="body1">{CHANGE_BIRTHDATE_CONTACT}</Typography>
+      <Section
+        className={classes.section}
+        title={CHANGE_GENDER}
+        content={CHANGE_GENDER_CONTACT}
+      />
+      <Section
+        className={classes.section}
+        title={CHANGE_BIRTHDATE}
+        content={CHANGE_BIRTHDATE_CONTACT}
+      />
       {isAccountInfoLoading ? (
         <CircularProgress />
       ) : accountInfoError ? (
         <Alert severity="error">{accountInfoError.message}</Alert>
       ) : (
         <>
-          <Username {...(accountInfo as GetAccountInfoRes.AsObject)} />
-          <Timezone {...(accountInfo as GetAccountInfoRes.AsObject)} />
-          <ChangeEmail {...(accountInfo as GetAccountInfoRes.AsObject)} />
-          <ChangePassword {...(accountInfo as GetAccountInfoRes.AsObject)} />
+          <Username
+            className={classes.section}
+            {...(accountInfo as GetAccountInfoRes.AsObject)}
```

Comment:
For future reference, we avoid spreading props since it's a bit opaque, and the type casting shows that typescript isn't being used very well here - but that wasn't your code and I don't want to be too annoying for your first contribution :)